packages/web/app/src/components/v2/modals/create-access-token.tsx

```diff
@@ -81,6 +92,137 @@ const CreateAccessTokenModalContent_OrganizationFragment = graphql(`
   }
 `);
 
+type TokenPreset = {
+  name: string;
+  description: string | ReactElement;
+  permissions: {
+    target: TargetAccessScope[];
+    project: ProjectAccessScope[];
+    organization: OrganizationAccessScope[];
+  };
+};
+
+const TOKEN_SIMPLE_PRESETS: TokenPreset[] = [
+  {
+    name: 'Schema Check & Push',
+    description:
+      'This set of permissions allows the token to check new schemas, push schemas, and report GraphQL operations usage.',
+    permissions: {
+      target: [TargetAccessScope.RegistryWrite],
+      project: [],
+      organization: [],
+    },
+  },
+  {
+    name: 'Schema Check Only',
+    description:
+      'This set of permissions allows the token to check new schemas. You can use this kind of token as part if your continuous integration pipeline.',
+    permissions: {
+      target: [TargetAccessScope.RegistryRead],
+      project: [],
+      organization: [],
+    },
+  },
+  {
+    name: 'GraphQL Operations Reporting',
+    description:
+      'This set of permissions allows the token to check new schemas, push schemas, and report GraphQL operations usage.',
+    permissions: {
+      target: [TargetAccessScope.RegistryWrite],
+      project: [],
+      organization: [],
+    },
+  },
+  {
+    name: 'Project Manager',
```

Comment:
@n1ru4l @dotansimha I'm not sure about `Project Manager`, `DevOps Engineer` and `Administrator`. 
I think these make sense for the `Members` page and not API tokens. Maybe let's use `Full access` instead of `Administrator` and drop the other two?

We could do `Project Manager` and `Administrator` as part of https://github.com/kamilkisiela/graphql-hive/issues/1157 for members of an organization.

WDYT?

Reply:
Yeah, agree!